Sunrise/ViewControllers/OrdersViewController.swift

```diff
@@ -11,7 +11,10 @@ import Result
 class OrdersViewController: UIViewController {
```

Comment:
Not a part of this PR, it'll be implemented for https://github.com/commercetools/commercetools-sunrise-ios/issues/89